sdcm/nemesis.py

```diff
@@ -2490,7 +2490,7 @@ def modify_table_compression(self):
         algo = random.choice(algos)
         prop_val = {"sstable_compression": algo}
         if algo:
-            prop_val["chunk_length_kb"] = random.choice(["4K", "64KB", "1M"])
+            prop_val["chunk_length_kb"] = random.choice(["4K", "64KB", "128KB"])
```

Comment:
Add to the commit message which nemesis you are fixing

Reply:
done